TypeCobol/Compiler/Preprocessor/ProcessedTokensDocument.cs

```diff
@@ -77,9 +77,14 @@ public IEnumerable<Token> ProcessedTokensSource
         /// - COPY directives text imports
         /// - REPLACE directive token remplacements
         /// </summary>
-        public static ITokensLinesIterator GetProcessedTokensIterator(TextSourceInfo textSourceInfo, ISearchableReadOnlyList<IProcessedTokensLine> lines, TypeCobolOptions compilerOptions)
+        /// <param name="textSourceInfo"></param>
+        /// <param name="lines"></param>
+        /// <param name="compilerOptions"></param>
+        /// <param name="allowWhitespaceTokens">This parameters is used to force whitspaces tokens to be also returned.</param>
+        /// <returns></returns>
+        public static ITokensLinesIterator GetProcessedTokensIterator(TextSourceInfo textSourceInfo, ISearchableReadOnlyList<IProcessedTokensLine> lines, TypeCobolOptions compilerOptions, bool allowWhitespaceTokens)
```

Comment:
The new `allowWhitespaceTokens` parameter is always `false` except for one usage, we could use a default value.

Reply:
I do that in order to not miss same places where it is called, but at the end, I think yes.